### PR TITLE
Autoupdater Rosetta2 detection on MacOs ARM

### PR DESCRIPTION
**Pull request by core1024**
Fixes https://github.com/Stremio/stremio-shell/issues/177

### CODE REVIEW

**Review by TRtomasz — APPROVED** [review]
Checked and work fine on arm

**Review by Ivshti — APPROVED** [review]

**Review by jaruba — APPROVED** [review]

**Comment by jaruba** [comment]
I don't fully understand the code but it follows the Apple suggested code to detect an app running in Rosetta2.

3 / 5 approvals should be enough for merging.

@core1024 Unless you have reasons to wait for more approvals, I think you can merge.

**Pull request merged**
